commit 8e8d4b20590547f5968314829118dbbbab078deb (refs/changes/17/3717/1)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-03-24T01:25:34+00:00 (1 year, 7 months ago)
    
    OS-6823 need uefi-csm-rom.bin with fix for OS-6604

diff --git a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
index 02f11ea5..0cbe839c 100755
Binary files a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin differ
